sor/src/main/java/com/bazaarvoice/emodb/sor/db/astyanax/RecordImpl.java

```diff
@@ -6,6 +6,7 @@
 import com.bazaarvoice.emodb.sor.db.Record;
 import com.bazaarvoice.emodb.sor.db.RecordEntryRawMetadata;
 
+import com.bazaarvoice.emodb.sor.db.test.DeltaClusteringKey;
 import java.util.Iterator;
 import java.util.Map;
 import java.util.UUID;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.UUID'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.qa.bazaarvoice.com/coding_rules#rule_key=squid%3AUselessImportCheck)